padguide2/padguide2.py

```diff
@@ -1109,6 +1109,18 @@ def __init__(self, m: PgMonster):
         self.color = [m.attr1.name.lower()]
         self.hascolor = [c.name.lower() for c in [m.attr1, m.attr2] if c]
 
+        
+        if m.limitbreak_stats:
+            self.hp = m.hp * m.limitbreak_stats
+            self.atk = m.atk * m.limitbreak_stats
+            self.rcv = m.rcv * m.limitbreak_stats
+            self.weighted_stats = m.weighted_stats * m.limitbreak_stats
+        else:
+            self.hp = m.hp if m.hp else None
+            self.atk = m.atk if m.atk else None
+            self.rcv = m.rcv if m.rcv else None
+            self.weighted_stats = m.weighted_stats if m.weighted_stats else None

```

Comment:
This code is weird. None of these values should ever be None, so I assume you're checking for false (0), but why would you replace 0 with None? 

Also, you're not doing the same thing for the limitbreak_stats section.

LMK if there is some good reason for this but I doubt it.

Also it would be more pythonic to replace this with 'self.hp = m.hp or None' etc (if there is a good reason for this code).